exporter/signalfxexporter/config.go

```diff
@@ -132,7 +131,7 @@ func (cfg *Config) validateConfig() error {
 
 func (cfg *Config) getIngestURL() (out *url.URL, err error) {
 	if cfg.IngestURL == "" {
-		out, err = url.Parse(fmt.Sprintf("https://ingest.%s.signalfx.com/v2/datapoint", cfg.Realm))
+		out, err = url.Parse(fmt.Sprintf("https://ingest.%s.signalfx.com", cfg.Realm))
```

Comment:
The documentation says:

>If a path is not included the exporter will automatically append the appropriate path, eg.: "v2/datapoint".

Will this still work?

Reply:
Yes, one of the unit tests proves it.